src/pages/TxSender.vue

```diff
@@ -24,10 +24,7 @@
 			<a-col :xs="{ span: 24 }" :lg="{ span: 12, offset: 2 }">
 				<h2>Transaction preview</h2>
 				<a-textarea
-					:style="
-						txOutput.length &&
-						'background-color: white !important; color: black !important'
-					"
+					class="text_area"
```

Comment:
I think you have removed the condition, this style should only be there if `txOutput.length`, please cross check it once

Reply:
yeah let me fix it!